baqend/getTestOverviewReport.ts

```diff
@@ -0,0 +1,189 @@
+import { EntityManager } from 'baqend';
+import { BulkTest, TestOverview } from 'baqend/commonjs/lib/model';
+import { Request, Response } from 'express';
+
+interface DomainStatus {
+  [status: string]: number;
+}
+
+interface SortedDomain {
+  domain: string;
+  count: number;
+  statuses: DomainStatus;
+}
+
+const EXCLUDED_DOMAINS = [
+  "kicker.de",
+  "plesk.com",
+  "speed-kit-test.com"
+];
+
+/**
+ * Extracts the origin from a given URL.
+ * 
+ * @param {string} url - The URL to extract the origin from.
+ * @returns {string} - The extracted origin.
+ */
+function extractOrigin(url: string): string {
```

Comment:
Sowas könnten wir auch in die `_helpers.ts` packen. Genaiu wie das `processOrigin`.